src/server/migrations/20210118154457_create_user_table.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('user', function(table) {
+    table.increments();
+    table.string('full_name').notNullable();
+    table.string('position');
+    table.string('linkedin');
+    table.string('github');
+    table.string('website');
+    table.string('profile_image_url');
+    table
+      .datetime('created_date')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+  });
+};
+
+exports.down = function(knex) {
+  return knex.schema.dropTable('user');
```

Comment:
**user** without -s also here

Reply:
i want to keep table name as user and not users. it would be better to change data in user controller file

Reply:
i will change in usercontroller .the table name from users to user.,. nice job Ember